lib/opencpu/client.rb

```diff
@@ -8,12 +8,29 @@ def initialize
     end
 
     def execute(package, function, options = {})
-      user   = options.fetch :user, :system
-      data   = options.fetch :data, {}
-      format = options.fetch :format, :json
-      github_remote = options.fetch :github_remote, false
+      user              = options.fetch :user, :system
+      data              = options.fetch :data, {}
+      format            = options.fetch :format, :json
+      github_remote     = options.fetch :github_remote, false
+      convert_na_to_nil = options.fetch :convert_na_to_nil, false
+
       process_query package_url(package, function, user, github_remote, :json), data, format do |response|
-        JSON.parse(response.body)
+        data =  JSON.parse(response.body)
+        data = convert_na_to_nil(data) if convert_na_to_nil
```

Comment:
Changed the variable name.

In the general case I would agree that using fetch for a value that defaults to falsey is redundant, but in this case it's more consistent with the way the rest of the options are evaluated.